crates/forge/src/init.rs

```diff
@@ -75,6 +117,17 @@ fn overwrite_files_from_scarb_template(
     Ok(())
 }
 
+fn create_deployment_script(base_path: &Path, project_name: &str) -> Result<()> {
+    let scripts_dir = base_path.join("scripts");
```

Comment:
This is not enough, it needs its' own package 

Please read this part for how it should look:
https://foundry-rs.github.io/starknet-foundry/starknet/script.html#1-scripts-directory-with-all-the-scripts-in-the-same-workspace-with-cairo-contracts-default-for-sncast-script-init